tests/functional/test_rose_stem.py

```diff
@@ -118,13 +117,12 @@ def mock_global_cfg(monkeymodule):
         conf: A fake rose global config as a string.
     """
     def _inner(target, conf):
-        """Mock a config object with a config node."""
-        node = ConfigLoader().load(StringIO(conf))
-
-        # Create a fake class imitating ConfigTree with .get_conf method:
-        config = MagicMock(spec=['get_conf'], get_conf=lambda: node)
+        """Get the ResourceLocator.default and patch its get_conf method
+        """
+        obj = ResourceLocator.default()
+        obj.get_conf = lambda: ConfigLoader().load(StringIO(conf))
```

Comment:
I think this will leave the patching in place after the test has run?

It would be safer to monkeypatch it
```suggestion
        monkeymodule.setattr(
            obj, 'get_conf', lambda: ConfigLoader().load(StringIO(conf))
        )
```